docs/docsource/operation/Metadata_Search/index.rst

```diff
@@ -0,0 +1,26 @@
+Metadata Search
+===============
+
+Production systems usually mark generated files with metadata tags.
+Scality’s object storage model also inscribes metadata to objects to
```

Comment:
inscribes = dedicates? Maybe it's a simpler word better understood by users?

Reply:
Changed to "assigns"